app/views/questions/_form.html.erb

```diff
@@ -1,21 +1,31 @@
 <%= form_tag(response_index_path, method: 'post', class: 'form-check', remote: true, id: 'response-form') do %>
   <div class="modal-body text-left" id='question-container'>
+    <% if alert %>
+      <!-- Alert if problem saving response. -->
+      <p class="alert alert-warning alert-dismissible fade show">
+        <%= alert %>
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </p>
+    <% end %>
     <% if @question.nil? %>
       <h5 class='card-title'>You finished the quiz!</h5>
     <% else %>
       <h5 class='card-title'><%= @question.title %></h5>
       <div id='options-container'>
         <% @question.options.each do |option_title, _| %>
-          <label><%= radio_button_tag(:selected_option, option_title) %> <%= option_title %></label>
+          <label><%= check_box_tag("selected_options[]", option_title) %> <%= option_title %></label>
```

Comment:
@kaandogusoy yeah, part of the changes for this feature involved changing radio to checkboxes here. This is necessary since there are multiple options that will lead to different categories/interests

Reply:
Actually I do believe they said that the quiz should be a mix of single-answer and multiple-answer questions, i.e. some radio and some checkbox.

Having it as all checkbox should in theory be a superset of necessary functions, but to make it look better perhaps add a simple boolean property to the question and toggle the radio/checkbox on the front-end .erb file?

Reply:
I think we should stick to "You Ain't Gonna Need It" on this one and not implement it. From what we've seen in the decision tree structure that the client provided, no questions will allow for a radio button option.

Reply:
Alright understood